src/applications/personalization/profile360/components/EmailSection.jsx

```diff
@@ -7,7 +7,7 @@ function EmailView({ data: emailData }) {
   return <span>{emailData.emailAddress}</span>;
 }
 
-export default function Vet360Email({ title = 'email address', fieldName = 'email', analyticsSectionName = 'email' }) {
+export default function Vet360Email({ title = 'Email address', fieldName = 'email', analyticsSectionName = 'email' }) {
   return (
```

Comment:
Not relevant to this PR, but once we start adding analytics I'm in favor of removing that `analyticsSectionName` and just using the `kebabCase` or some type of transformer on the field name.

Reply:
Yep I will check to see how the names line up